rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/ldap/LdapClient.java

```diff
@@ -187,15 +188,15 @@ public AccessIdRepresentationModel searchGroupByDn(final String name) {
     return accessId;
   }
 
-  public List<AccessIdRepresentationModel> searchGroupsofUsersIsMember(final String name)
+  public List<AccessIdRepresentationModel> searchGroupsAccessIdIsMemberOf(final String name)
```

Comment:
Does this work recursively?

Reply:
No, it doesn't. And based on my understanding, it shouldn't. Do you have another opinion?

Reply:
Was just wondering. Nothing more :)